radio/src/targets/sky9x/joystick.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) OpenTX
+ *
+ * Based on code named
+ *   th9x - http://code.google.com/p/th9x
+ *   er9x - http://code.google.com/p/er9x
+ *   gruvin9x - http://code.google.com/p/gruvin9x
+ *
+ * License GPLv2: http://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "opentx.h"
+
+#ifndef SIMU
+
+extern "C" {
+#include "usb/device/hid-joystick/HIDDJoystickDriver.h"
+extern void HIDDJoystickDriver_Initialize();
+unsigned char HIDDJoystickDriver_ChangeJoystickState(const HIDDJoystickInputReport *report);
+extern void USBD_Connect(void);
+}
+
+static void ConfigureUsbClock(void)
+{
+    /* Enable PLLB for USB */
+    PMC->CKGR_PLLBR = CKGR_PLLBR_DIVB(1)
```

Comment:
would you have a look at our indentation rules? we use 2 chars instead of 4 in OpenTX

Reply:
Please explain, do I need to use these rules only for code written by me, or do standard libraries also need to be reformed in accordance with these rules (for example, radio/src/targets/sky9x/usb/common/hid/HIDIdleRequest.c in this pull request)?

Reply:
Looks like the other ATMEL files were all left with original formatting, which makes sense in case there is a patch or other update (would simplify reviewing the diff).

@bsongis The existing ATMEL files are all 4-space indent.  Should you update the style guide to mention integrated vendor files? https://github.com/opentx/opentx/wiki/OpenTX-Code-Style-Guide

Reply:
I didn't say we do everything perfectly ;) I think this file (and massstorage.cpp file as well) should use our coding rules, as it is not exactly an ATMEL file. Otherwise yes, we try to change as little as possible fully imported code from other projects.

Reply:
Fixed in 777fff3